packages/developer-portal/src/components/pages/app-detail/app-content-v8.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from 'react'
```

Comment:
Can you put all the v8 app detail stuff in a v8 folder please? Just to make it easier to work out what's what when we want to switch over from the old pages to the new ones. Ta

Reply:
File missing tests